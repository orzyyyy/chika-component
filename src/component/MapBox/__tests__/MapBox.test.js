import React from 'react';
import { shallow } from 'enzyme';
let MapBox;
switch (process.env.LIB_DIR) {
  case 'lib':
    MapBox = require('../../../../lib/component/MapBox').default;
    break;
  case 'dist':
    MapBox = require('../../../../dist/lib/MapBox').default;
    break;
  default:
    MapBox = require('..').default;
    break;
}

describe('MapBox', () => {
  it('render correctly', () => {
    const wrapper = shallow(<MapBox />);
    expect(wrapper).toMatchSnapshot();
  });
});
