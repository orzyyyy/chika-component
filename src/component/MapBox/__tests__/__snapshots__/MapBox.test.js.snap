// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapBox render correctly 1`] = `
<div
  className="MapBox"
/>
`;
